DevelopersGuide.adoc

```diff
@@ -7427,6 +7427,37 @@ Basically you do not define an event map, and instead embed a handler in it's pl
                        ...)}}
 ```
 
+== Transitioning State
+
+There are two options for transitioning to another state from within a state:
+
+=== Option 1 -- Call `uism/activate` from within a handler
+
+```
+   ::uism/states
+     {:state/some-state
+       {::uism/events
+         {:event/some-event
+           {::uism/handler
+             (fn [env]
+               ,,, ; do some thing
```

Comment:
I know a lot of ppl use commas for ellipsis (I know there is a reason, but I don't remember why...code formatting?), but I'd prefer just an ellipsis, which is used everywhere else in the book..

Reply:
Oh yeah I get that.  I think the reason commas are used is because it's valid Clojure (commas are white space)

Reply:
yeah, just not valid English ;p

Reply:
since when is Clojure valid English ;)

Reply:
an ellipsis is the official english (and other language, now that I think about it) way to not have to say `;; and so on`....a bunch of commas strung together is gibberish in all languages.

:)

Reply:
Oh I remember...ppl do that so they can copy/paste colorized Clojure code from an editor without the commas being marked as syntax errors.